### PR TITLE
event-api feedback changes

### PR DESCRIPTION
**Pull request by SupriyaAdep**
Feedback mentioned in tickets https://agora.clickup.com/t/8556478/APP-3890

### CODE REVIEW

File: docs/first-party-extension/api-reference/events-library.md [hunk 1]
```diff
@@ -7,85 +7,94 @@ keywords: [MeetingInfoContextInterface, LayoutContextInterface]
...
-<br/>
-
-<collapsible>
-<method>
-
-### on: (evt: string, listener: [TEventCallback](#teventcallback) ) => void
-
-Subscribes to the event with the provided details.
+```ts
+send : (eventName: string, payload: string, persistLevel: EventPersistLevel ,receiver?: ReceiverUid ) => void
```
Comment (bhupendra1011) [inline docs/first-party-extension/api-reference/events-library.md:60]: please remove the code snippet and update the send method 
`### send:(eventName: string, payload: string, persistLevel:EventPersistLevel, receiver?: EventPersistLevel) => void`

File: docs/first-party-extension/api-reference/events-library.md [hunk 2]
```diff
@@ -95,133 +104,154 @@ Subscribes to the event with the provided details.
...
-</method>
-</collapsible>
+### on
 
-</method>
+Subscribes to the event. Use on method to add listener for specific event.
 
-</api>
+```ts
+on: (eventName: string, listener: Function ) => void
```
Comment (bhupendra1011) [inline docs/first-party-extension/api-reference/events-library.md:112]: same as above

File: docs/first-party-extension/api-reference/events-library.md [hunk 3]
```diff
@@ -95,133 +104,154 @@ Subscribes to the event with the provided details.
...
-| level?  | 2 \| 3 | Level of persistence     |
-| value   | string | Value to be presisted    |
+Additionally, method `on` returns `unbind` function. Call it and this listener
+will be removed from event.
 
-</method>
-
----
+```ts
+off: (eventName?: string, listener?: Function ) => void
```
Comment (bhupendra1011) [inline docs/first-party-extension/api-reference/events-library.md:164]: same as above

**Review by bhupendra1011 — CHANGES_REQUESTED** [review]

File: docs/first-party-extension/api-reference/events-library.md [hunk 4]
```diff
@@ -7,85 +7,92 @@ keywords: [MeetingInfoContextInterface, LayoutContextInterface]
...
+### send
 
-Subscribes to the event with the provided details.
+Sends the event with the provided details.
 
-| Prop     | Type                              | Description                                                       |
-| -------- | --------------------------------- | ----------------------------------------------------------------- |
-| evt      | string                            | Name of the event to be subscribed                                |
-| listener | [TEventCallback](#teventcallback) | Callback method for the event to be called when event is received |
+###send : (eventName: string, payload: string, persistLevel: EventPersistLevel ,receiver?: ReceiverUid ) => void
```
Comment (bhupendra1011) [inline docs/first-party-extension/api-reference/events-library.md:59]: there needs be a space between ### and rest of text and we can move it line no 55. it appears as below in preview : https://conferencing.agora.io/24010d60-26cf-4483-8aba-e7131687930a

Comment (Adictya) [reply docs/first-party-extension/api-reference/events-library.md:59]: Will fix after merge

**Pull request merged**